Protocol/ErrorMessages.xml

```diff
@@ -9090,6 +9090,28 @@
 							</ErrorMessage>
 						</ErrorMessages>
 					</Check>
+					<Check id="77">
+						<Name namespace="Protocol.Params.Param.ArrayOptions">CheckVolatileTableConsistency</Name>
+						<ErrorMessages>
+							<ErrorMessage id="1">
+								<Name>InvalidVolatileTableUsage</Name>
```

Comment:
InvalidVolatileTableUsage is missleading name (feels like you have to remove it) while we don't know if volatile is invalid or if another option (save, fk, dcf...) is invalid.
=> rebrand to IncompatibleVolatileOption